### ISSUE TITLE
Train from scratch, how to generate initial Neural Network

### ISSUE THREAD

**Issue by Griiimon** [lightvector/KataGo#197]
I like to train my own network from scratch. But i don't know how to generate the initial data needed by the selfplay/train/shuffle/gatekeep-loop. 
I figured i would need either a NN with random values based on some default model configuration or a set of matches (*.sgfs file). What is the first step here if i don't want to download any pre-trained networks?

**Comment by lightvector** [comment]
Just start the selfplay process. If it has no network then it will use a random number generator to play moves and start producing data.

**Comment by Griiimon** [comment]
Thanks! I think this is worth mentioning in the readme. Wasn't obvious to me even after browsing through the code..
